projects/components/src/table/table.component.scss

```diff
@@ -71,6 +76,14 @@ $header-height: 32px;
   }
 }
 
+.expandable-row {
```

Comment:
We need to test this on all pages a table is used. @Christian862 @itssharmasandeep could you guys please test this?